## Papers & Blog Posts on NLP for proteins

**ProtTrans: Towards Cracking the Language of Life's Code Through Self-Supervised Deep Learning and High Performance Computing**  
Ahmed Elnaggar, Michael Heinzinger, Christian Dallago, Ghalia Rihawi, Yu Wang, Llion Jones, Tom Gibbs, Tamas Feher, Christoph Angerer, Martin Steinegger, Debsindhu Bhowmik, Burkhard Rost  
_arXiv_, July 2020 | [Paper](https://arxiv.org/abs/2007.06225)

**BERTology Meets Biology: Interpreting Attention in Protein Language Models**  
Jesse Vig, Ali Madani, Lav R. Varshney, Caiming Xiong, Richard Socher, Nazneen Fatema Rajani  
_arXiv_, July 2020 | [Paper](https://arxiv.org/abs/2006.15222) | [Blog](https://blog.einstein.ai/provis/)

**USMPep: universal sequence models for major histocompatibility complex binding affinity prediction**  
Johanna Vielhaben, Markus Wenzel, Wojciech Samek & Nils Strodthoff  
_Bioinformatics_, July 2020 | [Paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03631-1)

**Transforming the Language of Life: Transformer Neural Networks for Protein Prediction Tasks**  
Ananthan Nambiar, Simon Liu, Mark Hopkins, Maeve Heflin,  View ORCID ProfileSergei Maslov, Anna Ritz  
_bioRxiv_, June 2020 | [Paper](https://doi.org/10.1101/2020.06.15.153643)

**ProGen: Language Modeling for Protein Generation**  
Ali Madani, Bryan McCann, Nikhil Naik, Nitish Shirish Keskar, Namrata Anand, Raphael R. Eguchi, Po-Ssu Huang, Richard Socher  
_bioRxiv_, March 2020 | [Paper](https://doi.org/10.1101/2020.03.07.982272) | [Blog](https://blog.einstein.ai/progen/)

**UDSMProt: universal deep sequence models for protein classification**  
Nils Strodthoff, Patrick Wagner, Markus Wenzel, Wojciech Samek  
_Bioinformatics_, January 2020 | [Paper]( https://doi.org/10.1093/bioinformatics/btaa003)  

**Modeling aspects of the language of life through transfer-learning protein sequences**  
Michael Heinzinger, Ahmed Elnaggar, Yu Wang, Christian Dallago, Dmitrii Nechaev, Florian Matthes, Burkhard Rost  
_BMC Bioinformatics_, December 2019 | [Paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3220-8)

**Generative models for graph-based protein design**  
John Ingraham, Vikas K. Garg, Regina Barzilay, Tommi Jaakkola  
_NeurIPS_, December 2019 | [Paper](https://papers.nips.cc/paper/9711-generative-models-for-graph-based-protein-design.pdf)

**Accurate Protein Structure Prediction by Embeddings and Deep Learning Representations**  
Iddo Drori, Darshan Thaker, Arjun Srivatsa, Daniel Jeong, Yueqi Wang, Linyong Nan, Fan Wu, Dimitri Leggas, Jinhao Lei, Weiyi Lu, Weilong Fu, Yuan Gao, Sashank Karri, Anand Kannan, Antonio Moretti, Mohammed AlQuraishi, Chen Keasar, Itsik Pe'er  
_arXiv_, November 2019 | [Paper](https://arxiv.org/abs/1911.05531)

**Athena: Automated Tuning of k-mer based Genomic Error Correction Algorithms using Language Models**  
Mustafa Abdallah, Ashraf Mahgoub, Hany Ahmed, Somali Chaterji  
_Scientific Reports_, November 2019 | [Paper](https://www.nature.com/articles/s41598-019-52196-4)

**Unified rational protein engineering with sequence-only deep representation learning**  
Ethan C. Alley, Grigory Khimulya, Surojit Biswas, Mohammed AlQuraishi, George M. Church  
Nature Methods, October 2019 | [Paper](https://www.nature.com/articles/s41592-019-0598-1)

**Evaluating Protein Transfer Learning with TAPE**  
Roshan Rao, Nicholas Bhattacharya, Neil Thomas, Yan Duan, Xi Chen, John Canny, Pieter Abbeel, Yun S. Song  
_bioRxiv_, June 2019 | [Paper](https://doi.org/10.1101/676825) | [Blog](https://bair.berkeley.edu/blog/2019/11/04/proteins/)

**Biological Structure and Function Emerge from Scaling Unsupervised Learning to 250 Million Protein Sequences**  
Alexander Rives, Siddharth Goyal, Joshua Meier, Demi Guo, Myle Ott, C. Lawrence Zitnick, Jerry Ma, Rob Fergus  
_bioRxiv_, May 2019 | [Paper](https://doi.org/10.1101/622803)

**A High Efficient Biological Language Model for Predicting Protein–Protein Interactions**  
Yanbin Wang, Zhu-Hong You, Shan Yang, Xiao Li, Tong-Hai Jiang, Xi Zhou  
_Cells_, February 2019 | [Paper](https://www.mdpi.com/2073-4409/8/2/122/htm)

**Learning continuous and data-driven molecular descriptors by translating equivalent chemical representations**  
Robin Winter, Floriane Montanari, Frank Noé, Djork-Arné Clevert  
_Chemical Science_, November 2018 | [Paper](https://doi.org/10.1039/C8SC04175J)

**Natural language processing in text mining for structural modeling of protein complexes**  
Varsha D. Badal, Petras J. Kundrotas, Ilya A. Vakser  
_BMC Bioinformatics_, March 2018 | [Paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2079-4)

**Identifying the missing proteins in human proteome by biological language model**  
Qiwen Dong, Kai Wang, Xuan Liu  
_BMC Systems Biology_, December 2016 | [Paper](https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-016-0352-6)

**Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics**  
Ehsaneddin Asgari, Mohammad R. K. Mofrad  
_PLOS ONE_, November 2015 | [Paper](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0141287)

**Survey of Natural Language Processing Techniques in Bioinformatics**  
Zhiqiang Zeng, Hua Shi, Yun Wu, Zhiling Hong  
_Computational and Mathematical Methods in Medicine_, October 2015 | [Paper](https://www.hindawi.com/journals/cmmm/2015/674296/)